### ISSUE TITLE
PARTNERSHIPS: Provide support and resources to staff on fundraising and grant management.

### ISSUE THREAD

**Issue by ldecoursy** [MozillaFoundation/Goals#14]
## GOALS
1. Complete series of training and information workshops
2. Facilitate monthly development calls
3. Create fundraising and grant management mana
4. Implement project design to grant management timeline with MLN staff
5. Identify and implement grant management system to meet growing grantmaking needs across MLN

**Comment by ldecoursy** [comment]
## MONTH
- ACCOMPLISHMENTS (What progress did we make towards these goals?)
- LEARNINGS (What challenges did we face/what went well/what did we learn?)
- PRIORITIES FOR NEXT MONTH (What are we focusing on next?)
- ON TRACK? BLOCKED? (List dependencies, or collaborations with other teams at MoFo/MoCo)

**Comment by ldecoursy** [comment]
@anmechung Can you please update progress towards our H2 goals using the template in my above comment? If better for someone else on your team let me know and I can ping them each month moving forward.

**Comment by anmechung** [comment]
Yes!  I'll be responsible for completing.  Question:  accomplishments, learnings up to now? i.e. H1 and H2? or in past month? i.e. August?

**Comment by ldecoursy** [comment]
@anmechung The idea is to update this ticket every month during H2. Should cover July and August for this one, and then I'll ping you monthly after that :)

**Comment by anmechung** [comment]
## JULY/AUGUST
- ACCOMPLISHMENTS 
  We completed: 1) first in-person MoFo and Money workshop in London followed by second workshop on grant writing in August; 2) Two monthly development calls (July and August); 3) launching mana 1.0;  4) implemented project design to grant management timeline along with project design tool in July; and 5) identified grant management system (Fluxx) currently under review by legal.
- LEARNINGS 
  Open and transparent processes and tools welcomed by staff, and critical for completing NSF proposal in short time frame.  When these processes and tools are utilized, the proposal development, approvals, and submission is more efficient, effective, and open for all involved.   Monthly development calls were very successful, and will move to bi-weekly calls beginning in September to more fully support program staff in fundraising. 
- PRIORITIES FOR NEXT MONTH 
- Getting all development staff to adhere to facilitating the utilization of processes and tools. 
- Continuing to update the mana
- Holding 3rd workshop on how to use Salesforce
- Initiating contract and plan to work with Fluxx on customize grantmaking system
- Working with operations team to assess evolving and complementary needs
- Finalizing Oct 10 Salon at Silicon Valley Community Foundation innovation conference
- ON TRACK? BLOCKED? 
  On track at the moment.  Legal needs to approve Fluxx as vendor before we can move forward.

**Comment by ldecoursy** [comment]
ping @anmechung for September update. Thanks!

**Comment by ldecoursy** [comment]
ping @anmechung Can you add an update for Sept/Oct using the same template as in the 2nd comment? Thanks!

**Comment by ldecoursy** [comment]
ping @anmechung Anything to add to wrap up this ticket (updates from Q4)? Going to add new tickets for each OKR for Jan.
